changelog/2022-03-01.md

```diff
@@ -1,13 +1,9 @@
-## :wrench: February 28, 2022
+## :wrench: March 1, 2022
 
 **CLI version: 0.8.1**
 
 Spring cleaning came early. This release features a lot of backend magic that improves overall stability and UX with Grid. We’re also excited to announce a dazzling set of enhancements to Datastores! **You’ll notice uploading to Datastores is now at least 5x faster!** 
-More details and information on how to use the feature are below.
-
-Upgrade your CLI to `v0.8.1` with `pip install lightning-grid --upgrade`  
```

Comment:
Why did we remove the pip install command?

Reply:
I think it is good to have it there.

Reply:
i removed it from here and added it to header.md, so it will always be fixed at the top of the release notes and we don't have to repeat it with every new post